cli/src/main.rs

```diff
@@ -91,6 +95,11 @@ enum Command {
         /// Additional derives
         #[structopt(long = "derive")]
         derives: Vec<String>,
+        /// Additional derives for a given type.
+        ///
+        /// Example `--derive_for_type my_module::my_type=serde::Serialize`.
+        #[structopt(long = "derive_for_type", parse(try_from_str = derive_for_type_parser))]
```

Comment:
Nit; can we go with `derive-for-type` please (ie dashes instead of underscores) :)